api/src/main/java/org/openmrs/module/pihcore/PihCoreUtil.java

```diff
@@ -37,7 +41,11 @@ public static String getDrugListChecksumDirectory() {
     }
 
     public static String getFormResource(String formName) {
-        return "file:" + getFormDirectory() + formName;
+        try {
+            return URLEncoder.encode("file:" + getFormDirectory() + formName, StandardCharsets.UTF_8.toString()).toString();
+        } catch (UnsupportedEncodingException e) {
+            return "file:" + getFormDirectory() + formName;
```

Comment:
Is there a use case here where this wouldn't be an error?  ie, would it be better just to throw an exception here?